tuhi/dbusserver.py

```diff
@@ -332,6 +336,34 @@ def __init__(self):
         self._is_searching = False
         self._searching_client = None
 
+    @GObject.Property
+    def is_searching(self):
+        return self._is_searching
+
+    @is_searching.setter
+    def is_searching(self, value):
+        if self._is_searching == value:
+            return
+
+        self._is_searching = value
+
+        props = GLib.VariantBuilder(GLib.VariantType('a{sv}'))
+        de = GLib.Variant.new_dict_entry(GLib.Variant.new_string('Searching'),
+                                         GLib.Variant.new_variant(
+                                             GLib.Variant.new_boolean(value)))
+        props.add_value(de)
+        props = props.end()
+        inval_props = GLib.VariantBuilder(GLib.VariantType('as'))
+        inval_props = inval_props.end()
+
+        self._connection.emit_signal(None, self.objpath,
+                                     "org.freedesktop.DBus.Properties",
+                                     "PropertiesChanged",
+                                     GLib.Variant.new_tuple(
+                                         GLib.Variant.new_string(INTF_MANAGER),
+                                         props,
+                                         inval_props))
```

Comment:
I think this boilerplate is quite common throughout the file now. Can't we just have a `_emit_properties_changed(props, inval_props)` function in this class?

Reply:
ftr, this is now somewhere in #34